python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -368,6 +368,59 @@ async def seeing_loop(self):
                 )
                 break
 
+    async def do_gotoAltAz(self)
+        """Move to Alt/AZ position.
+
+        Parameters
+        ----------
+        data : A SALOBJ data object
+            Contains the data as defined in the SAL XML file.
+        """
+        self.assert_enabled()
+        self.log.debug(
+            f"NOT YET IMPLEMENTED : Moving DIMM to azimuth"
+        )
```

Comment:
Please add `raise salobj.ExpectedError("Not implemented yet.")` to this and all other stubs.